qiskit_experiments/library/randomized_benchmarking/rb_experiment.py

```diff
@@ -214,25 +230,68 @@ def _generate_circuit(
                 circuits.append(rb_circ)
         return circuits
 
-    def _get_circuit_metadata(self, circuit):
-        if circuit.metadata["experiment_type"] == self._type:
-            return circuit.metadata
-        if circuit.metadata["experiment_type"] == ParallelExperiment.__name__:
-            for meta in circuit.metadata["composite_metadata"]:
-                if meta["physical_qubits"] == self.physical_qubits:
-                    return meta
-        return None
+    @classmethod
+    def _default_transpile_options(cls) -> Options:
+        options = super()._default_experiment_options()
+
+        # Assumes typical basis gates for superconduncting processor
+        options.basis_gates = ["sx", "rz", "x", "cx", "measure"]
+
+        return options
+
+    def _finalize(self):
+        super()._finalize()
+
+        # Set constraints to compute basis gates EPGs from an estimated EPC
+        gate_error_ratio = self.experiment_options.gate_error_ratio
+
+        if gate_error_ratio == "default":
```

Comment:
Instead of default I wonder if this should be "ibm_backend"` or something a bit more specific like that. There should probably also be a check that that all these gates are actually in the backends supported basis gates.

I think the default value should use the set backends basis gates in some way (maybe just set everything that isnt identity / rz to 1). So that this could work with my above example

Reply:
Good point. Probably
> maybe just set everything that isnt identity / rz to 1

is good direction.

Reply:
`lookup_epg_ratio` is added in https://github.com/Qiskit/qiskit-experiments/pull/762/commits/8db7f10250d0e0baff98340745bca379085b286b